policy/pa.go

```diff
@@ -174,24 +174,24 @@ func isDNSCharacter(ch byte) bool {
 
 var (
 	errInvalidIdentifier    = berrors.MalformedError("Invalid identifier type")
-	errNonPublic            = berrors.MalformedError("Name does not end in a public suffix")
-	errICANNTLD             = berrors.MalformedError("Name is an ICANN TLD")
-	errPolicyForbidden      = berrors.RejectedIdentifierError("Policy forbids issuing for name")
-	errInvalidDNSCharacter  = berrors.MalformedError("Invalid character in DNS name")
-	errNameTooLong          = berrors.MalformedError("DNS name too long")
-	errIPAddress            = berrors.MalformedError("Issuance for IP addresses not supported")
-	errTooManyLabels        = berrors.MalformedError("DNS name has too many labels")
-	errEmptyName            = berrors.MalformedError("DNS name was empty")
-	errNameEndsInDot        = berrors.MalformedError("DNS name ends in a period")
-	errTooFewLabels         = berrors.MalformedError("DNS name does not have enough labels")
-	errLabelTooShort        = berrors.MalformedError("DNS label is too short")
-	errLabelTooLong         = berrors.MalformedError("DNS label is too long")
-	errMalformedIDN         = berrors.MalformedError("DNS label contains malformed punycode")
-	errInvalidRLDH          = berrors.RejectedIdentifierError("DNS name contains a R-LDH label")
-	errTooManyWildcards     = berrors.MalformedError("DNS name had more than one wildcard")
-	errMalformedWildcard    = berrors.MalformedError("DNS name had a malformed wildcard label")
-	errICANNTLDWildcard     = berrors.MalformedError("DNS name was a wildcard for an ICANN TLD")
-	errWildcardNotSupported = berrors.MalformedError("Wildcard names not supported")
+	errNonPublic            = berrors.MalformedError("Domain name does not end with a valid public suffix (TLD)")
+	errICANNTLD             = berrors.MalformedError("Domain name is an ICANN TLD. Certificate can not be issued")
+	errPolicyForbidden      = berrors.RejectedIdentifierError("The ACME server can not issue a certificate for this domain name, because part of the name is forbidden by policy.")
```

Comment:
I still prefer "The ACME server refuses to issue for this domain name"

Reply:
Better. At least the user knows its a refusal not a syntax error or malformed string. But given the error itself is "errPolicyForbidden", surely its not unreasonable to add something like **"The ACME server refuses to issue for this domain name, because it is forbidden by policy"**, to point the way for the user, who needs some way to figure why it might be "refused".  

_(As an anecdote on this, the well known historical incident when users in the UK city of Scunthorpe were often blocked "by policy" because of the forbidden word in their domains, addresses, search strings etc.. So it's sometimes the case that a policy can cause unplanned isasues, and if so, the innocent user facing an unexpected "refusal" should have it made a little easier to figure why it has been "refused")_

Reply:
Sure, I'm happy with that.

Reply:
I've edited it. But on re-reading, the "it" seems just a bit ambiguous to me. Does it seem that way to you? 

_Refuses to issue a cert, because "it" is forbidden by policy_. 

But does that mean the domain name is forbidden/blacklisted?  Or the issue of any certificate to this entity? Or some other field in the request? Or the certificate issuance process overall (for some other reason)? Or what?  

It might seem obvious to us, but perhaps less so to an end user getting the message. Can you take a look at what's able to gives rise to this error, and perhaps suggest a pohrase that's even more specific?

Reply:
Going to leave this unless someone thinks otherwise, for simplicity.